### ISSUE TITLE
Debug mode not loading, 

### ISSUE THREAD

**Issue by freldelidp** [grokability/snipe-it#12530]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

While attempting to debug email-sending issues (Google), I hit a roadblock; Debug mode will not enable. I will include as much info as possible but I'm not really sure where to start:
laravel.log is not logging any issues (the log is working I tested by using an invalid APP_URL). I don't see anything in the apache logs (set at debug level).

In case it helps, when I try to send an Email (Gmail) I get a 500 error according to chrome. I also get mixed requests for HTTP://snipeit.lidp.com and https://snipeit.lidp.com, and I'm starting to wonder if my .env is not properly loading. 

### Reproduction steps

none?

### Expected behavior

a Dbug icon should appear in the lower left.

### Snipe-IT Version

v6.0.14 - build 9236 (master)

### Operating System

Ubuntu 22.04.1 LTS (LXC)

### Web Server

Apache2

### PHP Version

PHP 8.1.2-1ubuntu2.10

### Operating System

Windows 11

### Browser

Chrome 

### Version

110

### Device

Iphone 14

### Operating System

16.3.1

### Browser

Safari

### Version

latest

### Error messages

```shell
Not able to start Debug no Browser errors though. 

[PHP Modules]
bcmath
calendar
Core
ctype
curl
date
dom
exif
FFI
fileinfo
filter
ftp
gd
gettext
hash
iconv
json
ldap
libxml
mbstring
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
Phar
posix
readline
Reflection
session
shmop
SimpleXML
sockets
sodium
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
xml
xmlreader
xmlwriter
xsl
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache
```

### Additional context

Background:
This is a new install (v6.0.14 - install.sh)  on Ubuntu 22.04.1 LTS in an LXC container on Debian 11. A backup from our testing Container was restored after the new installation. This backup came from a Turnkey Linux LXC container running V5.something.

**Comment by snipe** [comment]
If you run `php upgrade.php`, what is the output?

**Comment by freldelidp** [comment]
@snipe 
[upgrade.php.txt](https://github.com/snipe/snipe-it/files/10760601/upgrade.php.txt)

I don't want to waste your time on this so if you think this is a nuke and pave situation let me know.

This may have also  contributed.
https://snipe.net/2017/11/15/so-you-ran-composer-as-root/




<blockquote><img src="https://snipe.net/wp-content/uploads/2017/11/maxresdefault-1024x576.jpg" width="48" align="right"><div>Snipe.Net</div><div><strong><a href="https://snipe.net/2017/11/15/so-you-ran-composer-as-root/">So you ran composer as root... - Snipe.Net</a></strong></div><div>Composer is a PHP dependency manager that’s used in just about any modern PHP application, and it works similarly to how Bundler works for Ruby. Even though Composer itself gives you a warning about not running it as root, lots of people disregard this warning and run it as root anyway. We run into this […]</div></blockquote>

**Comment by freldelidp** [comment]
I made a manual backup of snipe-it to prepare for a nuke & pave. I decided to explore this backup and found the .env file. This .env contained entirely different settings from what I had configured. I'm not sure what I did, but I seriously broke the backend somehow. It seems like some elements of snipe-it were using this configuration, and others were using the config I was editing.  `upgrade.php` had the info I needed to start asking the right questions. 

I am going to close this Bug because I think this was a PEBKAC issue.

Thanks again
